Gems/PhysX/Code/Source/ForceRegionForces.cpp

```diff
@@ -428,17 +428,29 @@ namespace PhysX
         entity.m_aabb.GetAsSphere(center, radius);
 
         const float crossSectionalArea = AZ::Constants::Pi * radius * radius;
-        const AZ::Vector3 velocitySquared = entity.m_velocity * entity.m_velocity;
+        const float speed = entity.m_velocity.GetLength();
+
+        if (speed < s_forceRegionMinSpeed)
+        {
+            return AZ::Vector3::CreateZero();
+        }
+
+        // Clamp the force to an upper limit which would correspond to stopping the entity in 0.2s. The time is somewhat arbitrary, but
+        // is intended to make sure that the velocity doesn't change so much in one timestep that oscillation could occur.
+        const float minTime = 0.2f;
```

Comment:
nit: From the description maybe we can surface this constant to the top of the file with the others for visibility:
```
static const s_minTimeForStoppingEntity = 0.2f; // Clamp the drag force to correspond to stopping an entity by this amount of time in seconds.
```

Reply:
Good idea, updated.